tests/integration/targets/dcnm_policy/tests/dcnm/dcnm_policy_merge.yaml

```diff
@@ -0,0 +1,516 @@
+##############################################
+##               SETUP                      ##
+##############################################
+
+- name: Put the fabric to default state
+  cisco.dcnm.dcnm_policy:
+    fabric: "{{ ansible_it_fabric }}" 
+    state: deleted                     # only choose form [merged, deleted, query]
+    config: 
+      - name: template_101  # name is mandatory
+      - name: template_102  # name is mandatory
+      - name: template_103  # name is mandatory
+      - name: template_104  # name is mandatory
+      - name: template_105  # name is mandatory
+      - switch:
+          - ip: "{{ ansible_switch1 }}"
+  register: result  
+
+- assert:
+    that:
+      - 'item["RETURN_CODE"] == 200'  
+  loop: '{{ result.response }}'
+
+
+- block:
+
+##############################################
+##                MERGE                     ##
+##############################################
+
+    - name: Create different non-existing policies 
```

Comment:
Just want to make sure I am reading this task correctly.

This will do the following:

* Create all policies on switch1
* If the IP on line 41 was changed to switch2
    * 104 and 105 would only be created on switch2
    * Would 101, 102 and 103 also be created on switch2 or would the policies under switch2 act as an override for the global policies?

Reply:
Yes, you are right. If policies are mentioned under a switch, then they will override all others. In this case as you said 104 and 105 will be created on switch2 and the other policies on switch1. But since I had a single switch I used switch1 again. But your understanding is right